spec.bs

```diff
@@ -1658,17 +1689,16 @@ table](https://fetch.spec.whatwg.org/#destination-table) to illustrate that <{fe
 A side effect of the fenced boundary model is that ads will lose the ability to know if a click
 resulted in a successful navigation. This is because the page loaded from a top-level [=navigate|
 navigation=] originating from a fenced frame will not be allowed to report to the fenced frame that
-it loaded (through something like <code>window.[=Window/opener=]</code>). Instead, we introduce a
-special event-level [=destination enum event/type|reporting type=], `reserved.top_navigation`, which
-automatically sends an [=report an event|event-level beacon=] when a fenced frame initiates a
-successful [=navigate|navigation=] to a [=top-level traversable=].
+it loaded (through something like <code>window.[=Window/opener=]</code>). Instead, we introduce
+special event-level [=destination enum event/type|reporting type=]s, `reserved.top_navigation_start`
```

Comment:
Right now the "reporting type" member is a string, but should it / could it be an [automatic beacon event type instead](https://pr-preview.s3.amazonaws.com/WICG/fenced-frame/pull/130.html#fencedframetype-automatic-beacon-event-type)? And please keep the code font when you link (see my last comment below in this review).